File: PO/speedtest/primes.js
const tab = [];
var n = 1e8;
for (let i = 2; i <= n; i++) {
    tab[i] = true;
}
var sum = 0;
for (let i = 2; i <= n; i++) {
    if (tab[i]) {
        for (let j = i + i; j <= n; j += i) {
            tab[j] = false;
        }
        sum += i;
    }
}
console.log(sum);

File: PO/speedtest/primes.cpp
const int n = 1e8;
bool tab[n + 1];

int main()
{
  for (int i = 2; i <= n; i++) {
    tab[i] = true;
  }
  long long sum = 0;
  for (int i = 2; i <= n; i++) {
    if (tab[i]) {
      sum += i;
      for (int j = i + i; j <= n; j += i) {
        tab[j] = false;
      }
    }
  }
  //printf("%lld", sum);
}
